src/elexmodel/handlers/data/Featurizer.py

```diff
@@ -33,27 +44,34 @@ def _center_features(self, df):
     def _add_intercept(self, df):
         df["intercept"] = 1
 
-    def _expand_fixed_effects(self, df, drop_first):
+    def _expand_fixed_effects(self, df: pd.DataFrame, drop_first: bool) -> pd.DataFrame:
```

Comment:
do you think it's worth adding this to all the functions? or slowly as we update code keep on adding?

Reply:
Ha, yeah the latter is what I was thinking and decided to just do that here. We could do a ticket at some point to add all the types but that would be a lot.

Reply:
that sounds good to me!